packages/react-core/src/layouts/Toolbar/ToolbarSection.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/layouts/Toolbar/styles.css';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Anything that can be rendered as toolbar section */
+  children: PropTypes.node,
+  /** Classes applied to toolbar section */
+  className: PropTypes.string,
+  /** Aria label applied to toolbar section */
+  'aria-label': props => {
+    if (!props['aria-label']) {
+      return new Error('ToolbarSection requires aria label to be specified');
```

Comment:
Could you change the warning to have the prop name as it would be used, 'aria-label' instead of 'aria label'.
Looks good otherwise!